src/DssSpell.t.sol

```diff
@@ -904,63 +908,346 @@ contract DssSpellTest is DssSpellTestBase {
+            );
+        }
+
+        // Old PSM is properly configured on AutoLine
+        {
+            (uint256 maxLine, uint256 gap, uint48 ttl, uint256 last,) = autoLine.ilks(SRC_ILK);
+            assertEq(maxLine, srcMaxLine, "after: AutoLine invalid maxLine");
+            assertEq(gap, srcGap, "after: AutoLine invalid gap");
+            assertEq(ttl, srcTtl, "after: AutoLine invalid ttl");
+            assertEq(last, block.number, "after: AutoLine invalid last");
+        }
+
+        // New PSM state is set correctly
+        {
+            // LitePSM ink is never modified
+            (uint256 dstInk, uint256 dstArt) = vat.urns(DST_ILK, address(dstPsm));
+            assertEq(dstInk, pdstInk, "after: unexpected dst ink chagne");
+            // There might be extra `art` because of the calls to `fill`.
+            assertGe(dstArt, pdstArt + expectedMoveWad, "after: dst art is not increased at least by the moved amount");
+            assertEq(dai.balanceOf(address(dstPsm)), dstBuf, "after: invalid dst psm dai balance");
+            assertEq(vat.gem(DST_ILK, address(dstPsm)), pdstVatGem, "after: unexpected dst vat gem change");
+            assertEq(
+                _amtToWad(gem.balanceOf(address(pocket))),
+                _amtToWad(pdstGemBalance) + expectedMoveWad,
+                "after: invalid gem balance for dst pocket"
+            );
+        }
+
+        // New PSM is properly configured on AutoLine
+        {
+            (uint256 maxLine, uint256 gap, uint48 ttl, uint256 last, uint256 lastInc) = autoLine.ilks(DST_ILK);
+            assertEq(maxLine, dstMaxLine, "after: AutoLine invalid maxLine");
+            assertEq(gap, dstGap, "after: AutoLine invalid gap");
+            assertEq(ttl, uint48(dstTtl), "after: AutoLine invalid ttl");
+            assertEq(last, block.number, "after: AutoLine invalid last");
+            assertEq(lastInc, block.timestamp, "after: AutoLine invalid lastInc");
+        }
     }
 
-    // RWA tests
-    function testRWA001Update() public {
-        RwaLiquidationOracleLike oracle = RwaLiquidationOracleLike(addr.addr("MIP21_LIQUIDATION_ORACLE"));
-        GemAbstract rwa001 = GemAbstract(addr.addr("RWA001"));
+    function test_CRON_LITE_PSM_JOB() public {
+        // ----- Pre-spell sanity checks -----
+
+        // Sequencer matches
+        assertEq(litePsmJob.sequencer(),     address(sequencer), "invalid sequencer");
+        // LitePsm matches
+        assertEq(litePsmJob.litePsm(),       address(dstPsm),    "invalid litePsm");
+        // fill: Set threshold at 15M DAI
+        assertEq(litePsmJob.rushThreshold(), rushThreshold,      "invalid rush threshold");
+        // trim: Set threshold at 30M DAI
+        assertEq(litePsmJob.gushThreshold(), gushThreshold,      "invalid rush threshold");
+        // chug: Set threshold at 300k DAI
+        assertEq(litePsmJob.cutThreshold(),  cutThreshold,       "invalid rush threshold");
+
+        // // ----- Execute spell -----
```

Comment:
Nit: double comments
```suggestion
        // ----- Execute spell -----
```
(here and below)

Reply:
fixed in [6090452](https://github.com/makerdao/spells-mainnet/pull/422/commits/609045268f815d407e2dc693ce72c606229ea88a)